modules/navier_stokes/include/fvkernels/INSFVMomentumTimeDerivative.h

```diff
@@ -23,4 +23,6 @@ class INSFVMomentumTimeDerivative : public INSFVTimeKernel
 protected:
   /// The density
   const Moose::Functor<ADReal> & _rho;
+  /// Whether to contribute to RC coefficients
+  const bool _contribute_to_rc_coeffs;
```

Comment:
I would move this into `INSFVTimeKernel` to avoid some duplication since both these classes you're modifying inherit from it

Reply:
done